rclcpp/include/rclcpp/node.hpp

```diff
@@ -270,6 +271,22 @@ class Node : public std::enable_shared_from_this<Node>
     const rclcpp::QoS & qos = rclcpp::ServicesQoS(),
     rclcpp::CallbackGroup::SharedPtr group = nullptr);
 
+  /// Create and return a Client.
+  /**
+   * \param[in] service_name The name on which the service is accessible.
+   * \param[in] qos Quality of service profile for client.
+   * \param[in] service_event_publisher_qos QOS profile for the service event publisher.
+   * \param[in] group Callback group to handle the reply to service calls.
+   * \return Shared pointer to the created client.
+   */
+  template<typename ServiceT>
+  typename Client<ServiceT>::SharedPtr
+  create_client(
+    const std::string & service_name,
+    const rclcpp::QoS & qos,
+    const rclcpp::QoS & service_event_publisher_qos,
+    rclcpp::CallbackGroup::SharedPtr group);
```

Comment:
IMO, these three parameters should be optional (i.e. assign some default values here).